service/src/main/java/bio/terra/cbas/dependencies/leonardo/AppUtils.java

```diff
@@ -93,24 +93,27 @@ ListAppResponse findBestAppForWds(List<ListAppResponse> apps)
                   var a = Objects.equals(app.getWorkspaceId(), wdsServerConfiguration.instanceId());
                   if (!a) {
                     logger.info(
-                        "Not using app {} for WDS because it is in workspace {}, not {}",
+                        "Not using app {} for {} because it is in workspace {}, not {}",
                         app.getAppName(),
+                        appType,
                         app.getWorkspaceId(),
                         wdsServerConfiguration.instanceId());
                   }
-                  var b = leonardoServerConfiguration.wdsAppTypes().contains(app.getAppType());
+                  var b = leonardoServerConfiguration.appTypeNames().contains(app.getAppType());
```

Comment:
When we add new app types for Workflows app and Cromwell runner app, does this line mean that we would be looking for WDS urls in these new app types as well?

Reply:
From my understanding, this function in getting a list of suitable apps from Leonardo. This line in particular I think is checking that the app that is currently being looked at is a valid app type in Leonardo.

Since WDS is now a stand-alone app, this change was mostly a naming covention change, as now we're getting an array of app types and not just a list of app types for the sole purpose of getting the wds proxy url.

As far as I'm aware, the only way to get the WDS urls will be from the wds app type. The new app types won't have any knowledge of wds urls like the cromwell app type does today.